tests/ethereum/test_general.py

```diff
@@ -1774,6 +1778,7 @@ def test_gas_check(self):
         except TerminateState as e:
             result = str(e)
         self.assertEqual(result, "SELFDESTRUCT")
+        consts.oog = default_gas
```

Comment:
This is possible going to fail if the tests get executed concurrently. Config must go with mcore instance and be frozen and etc etc.
Also there this 
https://github.com/trailofbits/manticore/blob/c8e5e25d8e10c6235bd18fec2d3d44b56383c58a/manticore/utils/config.py#L73-L80